pkg/deployment/images.go

```diff
@@ -167,6 +167,8 @@ func (ib *imagesBuilder) fetchArangoDBImageIDAndVersion(ctx context.Context, ima
 	args := []string{
 		"--server.authentication=false",
 		fmt.Sprintf("--server.endpoint=tcp://[::]:%d", k8sutil.ArangoPort),
```

Comment:
does this imply that the connection will never be ssl?  and never use authentication?

Reply:
For the ID pod : Yes.

Reply:
The ID pod is run very quickly to fetch version information for a given image name.

Reply:
It will never be accessible from outside, nor will it store any data or live longer than a few seconds